evalml/model_understanding/graphs.py

```diff
@@ -947,9 +1034,29 @@ def graph_partial_dependence(
                     x=part_dep["feature_values"],
                     y=part_dep["partial_dependence"],
                     name="Partial Dependence",
-                    line=dict(width=3),
+                    line=dict(width=3, color="rgb(99,110,250)"),
```

Comment:
The RGB number corresponds to the default blue that I extracted from the default MacOS color picker. I don't think there's like an alias to this exact color in plotly, but I could be missing it from the docs